docker-compose-starburst.yml

```diff
@@ -16,45 +16,13 @@ services:
       POSTGRES_USER: dbt-trino
       POSTGRES_PASSWORD: dbt-trino
 
-  metastore_db:
-    image: postgres:11
-    hostname: metastore_db
-    environment:
-      POSTGRES_USER: hive
-      POSTGRES_PASSWORD: hive
-      POSTGRES_DB: metastore
-
   hive-metastore:
+    image: ghcr.io/innoverio/hive-metastore-multiarch:sha-28c28bc #TODO: replace with Starburst hive container once config is merged
```

Comment:
Which config is merged? Are you referring to Starburst HMS?

Reply:
From our perspective it is much easier to have full access to the hive config instead of the 2-step approach in the Starburst HMS (you set up some env variable and then do something with that env variable). Also it would mean that every test scenario we imagine now also becomes supportable with the Starburst HMS (is that something we want?).